core/domain/platform_parameter_domain_test.py

```diff
@@ -25,7 +25,7 @@
 from core.domain import platform_parameter_domain as parameter_domain
 from core.tests import test_utils
 
-SERVER_MODES = parameter_domain.SERVER_MODES # pylint: disable=invalid-name
+SERVER_MODES = parameter_domain.ServerModes # pylint: disable=invalid-name
```

Comment:
Can we be consistent? Should we change the variable name here?

Reply:
Yes

Reply:
@nithusha21 as per offline discussion with @vojtechjelinek, we decided to rename variable name as well, as task is divided into so small parts, we should go with correct variable name and reduce pylint error as much as possible.

Updating issue description accordingly.

Thanks.

Reply:
Should I update all of them???
Also, according to me there are two such variable names in this PR; **SERVER_MODES** and **FEATURE_STAGES** which needs to be updated to ensure consistency. Should I go ahead and change them @vojtechjelinek @nithusha21 @JeeveshGarg ???

Reply:
@AyushJ7 yes change them too, and read issue description again, it is updated now. Clearly mention answer of your query.
Thanks.

Reply:
Done!!!
Thanks for clarification!!!